### PR TITLE
feat(schematic): added component requirement endpoints

### PR DESCRIPTION
**Pull request by andrewelamb**
- fixes [FDS-829](https://sagebionetworks.jira.com/browse/FDS-829?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ)
- Adds component requirements endpoints (one for array and one for graph)

Check the following files (The rest are auto-generated and can be ignored for code review):

- all yaml files in libs/schematic/api-description/src/
- apps/schematic/api/schematic_api/controllers/schema_controller_impl.py
- apps/schematic/api/schematic_api/test/test_schema_controller_impl.py
- The two python files are the most important to look at

[FDS-829]: https://sagebionetworks.jira.com/browse/FDS-829?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/schematic/api/schematic_api/test/test_manifest_generation_controller_impl.py [hunk 1]
```diff
@@ -45,92 +46,7 @@ def test_error_statuses(self, test_schema_url: str) -> None:
                     manifest_title="title",
                     use_strict_validation=True,
                     generate_all_manifests=False,
+                    display_label_type="class_label",
                 )
                 assert status == 404
                 assert isinstance(result, BasicError)
-
```
Comment (mialy-defelice) [inline apps/schematic/api/schematic_api/test/test_manifest_generation_controller_impl.py:51]: Why were these removed?

**Review by mialy-defelice — APPROVED** [review]

File: apps/schematic/api/schematic_api/test/test_manifest_generation_controller_impl.py (hunk 1 above)
Comment (andrewelamb) [reply apps/schematic/api/schematic_api/test/test_manifest_generation_controller_impl.py:51]: I'm not sure why those are showing up. Everythgin to do with teh google sheets endpoint is from a previous PR. I don't think the github files changed viewer is handling the 

"@andrewelamb
[Merge branch 'main' into fetaure-FDS-829-component-requirements](https://github.com/Sage-Bionetworks/sage-monorepo/pull/2667/commits/8693f40c5276cd91777fc5f4efc59a15cb81f9f7)"

correctly.
